x-pack/legacy/plugins/apm/public/components/app/APMIndicesPermission/index.tsx

```diff
@@ -96,12 +111,30 @@ const PermissionPage = ({ onEscapeHatchClick }: Props) => {
                 </h2>
               }
               body={
-                <p>
-                  {i18n.translate('xpack.apm.permission.description', {
-                    defaultMessage:
-                      "We've detected your current role in Kibana does not grant you access to the APM data. Please check with your Kibana administrator to get the proper privileges granted in order to start using APM."
-                  })}
-                </p>
+                <>
+                  <p>
+                    {i18n.translate('xpack.apm.permission.description', {
+                      defaultMessage:
+                        'We’ve detected your current role in Kibana does not grant you access to the APM data. Please check with your Kibana administrator to get the proper privileges granted to start using APM.'
+                    })}
+                  </p>
+                  <>
+                    <EuiText size="s">
+                      <span style={{ fontWeight: 'bold' }}>
+                        {i18n.translate('xpack.apm.permission.missingIndices', {
+                          defaultMessage: 'Missing indices permissions:'
```

Comment:
Perhaps we can leave this out if we modify the copy above?

Reply:
@formgeist  WDYT?

Reply:
SGTM